sport/app/football/feed/agent.scala

```diff
@@ -154,10 +154,30 @@ class CompetitionAgent(
         }
       }
 
-      //it is important that newMatches are at the start of the list here
-      comp.copy(matches = (newMatches ++ comp.matches).sorted(MatchStatusOrdering).distinctBy(_.id).sortByDate)
+      val allMatches = (newMatches ++ comp.matches).sorted(MatchStatusOrdering).distinctBy(_.id).sortByDate
+
+      val updatedMatches = allMatches.filter(footballMatch =>
+        (!isPlaceholderMatch(footballMatch)) ||
+          (isPlaceholderMatch(footballMatch) && !nonPlaceHolderMatchWithSameDateExists(allMatches, footballMatch)),
+      )
+
+      comp.copy(matches = updatedMatches)
     }
 
+  def isPlaceholderMatch(footballMatch: FootballMatch): Boolean = {
+    val placeholderIndicator = Set("Winner", "Runner-up", "Wnr Gp", "R-Up Gp", "Loser")
```

Comment:
Created this set from what I can see here and previous cases:

<img width="932" alt="image" src="https://user-images.githubusercontent.com/19683595/205672894-74cd3af9-6f22-4414-8560-39c6f24e8aa1.png">